web/angular-wallet/src/app/main/dashboard/balance-chart/balance-chart.component.ts

```diff
@@ -81,7 +81,8 @@ export class BalanceChartComponent extends UnsubscribeOnDestroy implements OnIni
           fill: 'start'
         }
       ],
-      labels: this.balanceHistory.map(({timestamp}) => this.toDateString(convertBurstTimeToDate(timestamp))),
+      labels: this.balanceHistory.map(({timestamp}) => timestamp && this.toDateString(convertBurstTimeToDate(timestamp)) ||
+      this.toDateString(new Date())),
```

Comment:
Since BurstJS 0.5.0 we have some useful Value Objects. `BurstTime` is one of them
https://burst-apps-team.github.io/phoenix/classes/util.bursttime.html - no need to change, but in new projects worth being considered

The question is: how can we have tx without a timestamp...weird.

Reply:
I think it was a stuck/orphan transaction resulting from fork testing.